docs/json/radarr/cf/atmos-undefined.json

```
@@ -71,6 +71,15 @@
         "value": "\\b(l?)PCM(\\b|\\d)"
       }
     },
+    {
+      "name": "Not RlsGrp (Atmos Only)",
+      "implementation": "ReleaseTitleSpecification",
+      "negate": true,
+      "required": true,
+      "fields": {
+        "value": "W4NK3R|HQMUX"
```

wouldn't `-DON` be needed here as well?
```suggestion
        "value": "\\b(W4NK3R|HQMUX|-DON)\\b"
```

---

I just restored this line back to it's previous state, so I don't think HQMUX is needed.

---

Same issue with the `\b`

---

Sorry - typed the wrong thing hence the comment deletions.

I restored this line to it's previous state. Don't think `-DON` is needed as I don't believe they're naming releases as Atmos undefined when actually they're THD Atmos.